### PR TITLE
Updating Checkbox, Avatar, Button

### PR DESCRIPTION
**Pull request by Duncid**
- Adding new styles to Checkox for Tree
- Changing the size scale of Avatar
- Creating an Avatar variant of button
- First draft of a Tree component in Sparkle

## BEWARE

- Breaking change on Avatar
- Tree component is not functional. Structure is there, need to port the logic.

### CODE REVIEW

File: sparkle/src/stories/Tree.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import { Tree } from "../index_with_tw_base";
+
+const meta = {
+  title: "Atoms/Tree",
+  component: Tree,
+} satisfies Meta<typeof Tree>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
```
Comment (spolu) [inline sparkle/src/stories/Tree.stories.tsx:12]: unused :grimacing:

Comment (Duncid) [reply sparkle/src/stories/Tree.stories.tsx]: Removed

**Comment by spolu** [comment]
LGTM but would love @fontanierh eyes on the Tree component to make sure it'll be extendable to how we use it :+1:

**Comment by Duncid** [comment]
IMO no need to review as @fontanierh will probably carry out the rest later on!
Ideally @philipperolet would merge and manage migration of Avatar as he needs the Button variant avatar to complete suggest.

**Comment by fontanierh** [comment]
tree will likely require some extra but definitely not blocking to merge 👍

**Comment by philipperolet** [comment]
Very good then :)

**Pull request merged**